src/applications/health-care-supply-reordering/tests/mdot.cypress.spec.js

```diff
@@ -34,6 +34,7 @@ const testConfig = createTestConfig(
             cy.findByText('Add a temporary address', {
               selector: 'button',
             }).click({ force: true });
+            cy.findByLabelText('State').select('Puerto Rico');
```

Comment:
hmm, does selecting puerto rico and canada together work? 🤔

Reply:
Selecting a different country clears the State dropdown (and if changing to country other than US, changes dropdown to an input field.

Reply:
And yes, dropping this one-liner in was my way of testing the functionality instead of writing and entirely new test :-)

Reply:
ah, got it!

![gif](https://media.tenor.com/zd1jUmlynEIAAAAM/alright-carry-on.gif)